src/main/webapp/app/js/goal-page.js

```diff
@@ -69,9 +72,67 @@ function addComment() {
                 document.getElementById("userComment").value = "";
                 onLoad();
             } else {
-                alert("Sompthing went wrong! error:404");
+                alert("Something went wrong! error:404");
```

Comment:
Please append the actual `response.status` instead of `404`, or remove mention of status code.